### PR TITLE
fix(client): center executing booking pill

### PR DESCRIPTION
**Pull request by mattydono**
- Centered executing booking pill in analytics

### CODE REVIEW

File: src/client/src/apps/MainRoute/widgets/spotTile/components/notifications/TileBooking.tsx [hunk 1]
```diff
@@ -32,7 +32,7 @@ const BookingPill = styled.div<{
     isAnalyticsView && !isExecutingStatus ? '3.5px' : isExecutingStatus ? '10px' : '5px'};
   position: absolute;
   ${({ isAnalyticsView, isExecutingStatus }) =>
-    isAnalyticsView && !isExecutingStatus ? 'right: 1.5rem' : isAnalyticsView ? 'left: -20%' : ''};
+    isAnalyticsView && !isExecutingStatus ? 'right: 1.5rem' : isAnalyticsView ? 'top: 31.5px' : ''};
```
Comment (rikoe) [inline src/client/src/apps/MainRoute/widgets/spotTile/components/notifications/TileBooking.tsx]: @mattydono - does this top margin work correctly under all sizes of the responsive app, given that it is not relative anymore... please check small, large and mobile.

Comment (rikoe) [reply src/client/src/apps/MainRoute/widgets/spotTile/components/notifications/TileBooking.tsx]: Oh and screenshot/gif please

Comment (mozmorris) [reply src/client/src/apps/MainRoute/widgets/spotTile/components/notifications/TileBooking.tsx]: I'd suggest using/checking Storybook when making this type of change. It makes testing various states and layouts a breeze (and quickly identifying problems 😉)

Comment (mattydono) [reply src/client/src/apps/MainRoute/widgets/spotTile/components/notifications/TileBooking.tsx]: @rikoe what screen sizes do we support down to?

Comment (rikoe) [reply src/client/src/apps/MainRoute/widgets/spotTile/components/notifications/TileBooking.tsx]: Apple watch! 😂 

Mobile is probably smallest, but the spot tile has a minimum size, so you can only go so small.

File: src/client/src/apps/MainRoute/widgets/spotTile/components/PriceControls/styled.tsx [hunk 2]
```diff
@@ -4,7 +4,6 @@ export const PriceControlsStyle = styled('div')<{ isAnalyticsView: boolean }>`
   display: flex;
   justify-content: space-between;
   ${({ isAnalyticsView }) => (isAnalyticsView ? `` : `align-items: center; margin-top: 15px;`)}
-  position: relative;
```
Comment (mozmorris) [inline src/client/src/apps/MainRoute/widgets/spotTile/components/PriceControls/styled.tsx]: Screenshot 2020-01-30 at 10 41 16

This has an affect elsewhere 🕵

**Pull request merged**